client/sites-dashboard/components/sites-ellipsis-menu.tsx

```diff
@@ -176,6 +180,32 @@ const PreviewSiteModalItem = ( { recordTracks, site }: SitesMenuItemProps ) => {
 	);
 };
 
+const CopySiteItem = ( { recordTracks, site }: SitesMenuItemProps ) => {
+	const { __ } = useI18n();
+	const hasAtomicFeature = useSafeSiteHasFeature( site.ID, WPCOM_FEATURES_ATOMIC );
+	const userId = useSelector( ( state ) => getCurrentUserId( state ) );
+	const plan = site.plan;
+	const isSiteOwner = site.site_owner === userId;
+
+	if ( ! hasAtomicFeature || ! isSiteOwner || ! plan ) {
+		return null;
+	}
+
+	const copySiteHref = addQueryArgs( `/setup/copy-site`, {
+		productSlug: plan.product_slug,
+		sourceSlug: site.slug,
+		sourceUrl: site.URL,
+	} );
+	return (
+		<MenuItemLink
+			href={ copySiteHref }
+			onClick={ () => recordTracks( 'calypso_sites_dashboard_site_action_copy_site' ) }
```

Comment:
Let's get this registered with Tracks before merge.

Reply:
Registered 1330-gh-Automattic/tracks-events-registration